packages/api/src/controllers/asset.test.ts

```diff
@@ -316,5 +324,58 @@ describe("controllers/asset", () => {
         },
       });
     });
+    describe("chunked upload", () => {
+      jest.setTimeout(10000);
+      it("should start upload, stop it, resume it on tus test server", async () => {
```

Comment:
This test is not really doing what it describes. It's missing some logic to interrupt the upload and then resuming it from where it stopped (the interrupt/resume logic are there but are never executed, just declared as a lambda that is never called).

It's also not really testing if the uploads work, it's only checking that the `tus` client finishes cleanly. You should also check if the files are actually uploaded to where they should, ideally ensuring that it's named after the asset playback ID as well and that the `task` gets scheduled after the uploads is complete (you get the `task` ID on the `/request-upload` response, just need to check if it's `state.phase` changes to `waiting` (from `pending`))